### ISSUE TITLE
[Feature request] Ability to copy stream/portion of stream

### ISSUE THREAD

**Issue by opusforlife2** [mifi/lossless-cut#284]
Uh, I know the software is called lossless-cut and all... but Audacity is the only music editing software I know where I can double the length of a track and it doesn't support the Opus codec!

Anyway, if this is something you'd be interested in, my use case is simple. I have some tracks where I want to loop them once or twice, doubling or tripling their length. So I need to copy the entire track and paste it ahead of itself.

**Comment by mifi** [comment]
If I understand correctly you can do this with lossless cut. Just duplicate the file and then use the merge function from "tools" to merge those two files

**Comment by opusforlife2** [comment]
Oh... Ummm... Yeah. That works. I should have thought of that. Thanks.

**Comment by markusd1984** [comment]
@mifi FYI - I just had similar use case but including video. Basically I wanted to do a quick and dirty edit to cut some parts of a home workout video and duplicate some parts, thus I did use your suggested approach exporting segments to then duplicate the desired segment files as many as I needed, for which I ended up creating also labels as I started mixing up some of the files and thus didn't achieve the final end results of file orders and segment repetitions.

Hence I was thinking to have a duplicate function within LC could be a nice to have: where one can select an segment, duplicate it and perhaps even **arrange placement within the timeline.**, which would also have the benefit of being able to let some parts later in the video to appear actually earlier (for those who don't need duplication), again all without having to export and arrange file orders. 😄

Two nice to have features but perhaps a quick/easier way to implement similar to simplify the workflow would be in the "**Sort your files for merge"** dialogue to have a duplicate option there, so one at least doesn't has to duplicate the files prior.
